modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/typescriptnode/TypeScriptNodeClientCodegenTest.java

```diff
@@ -296,6 +296,26 @@ public void testDeprecatedAttribute() throws IOException {
                 "* @deprecated");
     }
 
+    @Test
+    public void testDeprecatedOperation() throws IOException {
+
+        File output = Files.createTempDirectory("typescriptnodeclient_").toFile();
+        output.deleteOnExit();
+
+        final CodegenConfigurator configurator = new CodegenConfigurator()
+                .setGeneratorName("typescript-node")
+                .setInputSpec("src/test/resources/3_0/typescript-node/SampleProject.yaml")
+                .setOutputDir(output.getAbsolutePath().replace("\\", "/"));
+
+        final ClientOptInput clientOptInput = configurator.toClientOptInput();
+        DefaultGenerator generator = new DefaultGenerator();
+        List<File> files = generator.opts(clientOptInput).generate();
+        //files.forEach(File::deleteOnExit);
+
+        TestUtils.assertFileContains(Paths.get(output + "/api/defaultApi.ts"),
+                "* @deprecated", 118);
```

Comment:
asserting on the line number feels very fragile to me. Any refactor in the mustache that changes any lines before the deprecated tag will require future contributors to update this assertion.

Reply:
Agree. Now that I think about it, it isn't a great idea. I have removed it.